ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,8 @@
-### What did you do?
-(Please include as many details as possible.)
+### Describe the problem and steps to reproduce it:
```

Comment:
Yeah, the only issue I see is that `Describe the problem...` and `What happened` are a bit overlapping.